.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+
+#### Contributing
+If you would like to contribute to hazelcast-go-client please fork, fix, commit and send a pull request. Your code needs
```

Comment:
I think these steps should be easy to follow for an external person and be in complete sentences. Below is my try
```
If you would like to contribute to hazelcast-go-client;
- Fork the original repo
- Commit your contributions in a new branch
- Send a Pull Request

For detailed instructions for our git process: [Developing with Git](https://hazelcast.atlassian.net/wiki/display/COM/Developing+with+Git)

In order to merge your pull request, we need the contributor to sign [Hazelcast Contributor Agreement](https://hazelcast.atlassian.net/wiki/display/COM/Hazelcast+Contributor+Agreement). You do not have to wait for approval for doing this step.

Please make sure that your code is formatted using `gofmt`
Please make sure that your code compiles and passes all the tests by running `build.sh` before sending a pull request.